fe/fe-core/src/main/java/com/starrocks/analysis/SelectStmt.java

```diff
@@ -1233,18 +1233,18 @@ private ExprSubstitutionMap createSumOrCountMultiDistinctSMap(
             final String functionName = inputExpr.getFnName().getFunction();
             if (functionName.equalsIgnoreCase(FunctionSet.COUNT)) {
                 final List<Expr> countInputExpr = Lists.newArrayList(inputExpr.getChild(0).clone(null));
-                replaceExpr = new FunctionCallExpr("MULTI_DISTINCT_COUNT",
+                replaceExpr = new FunctionCallExpr("multi_distinct_count",
                         new FunctionParams(inputExpr.isDistinct(), countInputExpr));
             } else if (functionName.equalsIgnoreCase("SUM")) {
                 final List<Expr> sumInputExprs = Lists.newArrayList(inputExpr.getChild(0).clone(null));
-                replaceExpr = new FunctionCallExpr("MULTI_DISTINCT_SUM",
+                replaceExpr = new FunctionCallExpr("multi_distinct_sum",
```

Comment:
why change to lower case

Reply:
All function name should be lower case

Reply:
PhysicalHashAggregateOperator::couldApplyLowCardAggregateFunction was lower case

Reply:
I'll use FunctionSet.MULTI_DISTINCT_SUM instead of multi_distinct_sum